src/graphql/resolvers/global-descriptor-resolvers.ts

```diff
@@ -0,0 +1,160 @@
+import { IResolvers } from 'graphql-tools';
+import services from '../../services';
+import { IContext } from '../context';
+import { PubSub } from 'graphql-subscriptions';
+import { GlobalDescriptorService } from '../../services/global-descriptor-service';
+
+export interface IDescriptor {
+	unit: String;
+	value: number;
+	longitude: number;
+	latitude: number;
+	unixTimestamp: number;
+}
+
+export enum TRANSACTION_TYPE {
+	TRANSACTION_HASH,
+	RECIEPT,
+	CONFIRMATION,
+	ERROR
+}
+/*
+interface IServiceMap {
+	globalDescriptorService: GlobalDescriptorService;
+	
+}
+class service{
+	 public serviceMap: IServiceMap = {
+		globalDescriptorService: new GlobalDescriptorService(),

```

Comment:
You can do the following:
```
const globalDescriptors = new GlobalDescriptorService();
```
instead of calling service map, and then just remove any reference to service map.